_po/ja/how-to/laravel.po

```diff
@@ -193,38 +264,57 @@ msgid ""
 "            return static::query();\n"
 "        }"
 msgstr ""
+"    public static function search($keyword)\n"
+"    {\n"
+"        if(empty($keyword)){\n"
+"            return static::query();\n"
+"        }"
 
 msgid "        $search_columns = ['title', 'body'];"
-msgstr ""
+msgstr "        $search_columns = ['title', 'body'];"
 
 msgid "        $search_query = static::query();"
-msgstr ""
+msgstr "        $search_query = static::query();"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.